app/models/section.rb

```diff
@@ -112,7 +116,38 @@ def self.from_publishing_api(content_id:)
     section
   end
 
-private
+  def updated_manual_links?
```

Comment:
I added in the question mark because it returns a boolean, but looking at it again, the method #save is a similar pattern. I will change it to "update_manual_links"